qemu/tests/nvdimm.py

```diff
@@ -158,6 +159,15 @@ def run(test, params, env):
         nvdimm_test.verify_nvdimm(vm, mems)
         error_context.context("Format and mount nvdimm in guest", logging.info)
         nvdimm_test.mount_nvdimm()
+        if params.get("nvml_test", "no") == "yes":
+            pkgs = params["depends_pkgs"].split()
+            pkgmgr = utils_package.RemotePackageMgr(nvdimm_test.session, pkgs)
+            if not pkgmgr.install():
+                test.cancel("Install dependency packages failed")
```

Comment:
```
if not utils_package.package_install(pkgs, nvdimm_test.session):
    test.cancel(***)
```

Reply:
Could you pls explain why it is better to invoke package_install?

Reply:
> Could you pls explain why it is better to invoke package_install?

More compact, 2 line is enough.

Reply:
Updated, pls help review again, thanks.